### PR TITLE
Connecting join functionality

### PR DESCRIPTION
**Pull request by andru47**

### CODE REVIEW

File: src/main/java/com/google/sps/servlets/Utils.java [hunk 1]
```diff
@@ -96,6 +96,9 @@ static void joinGCalendarEvent(String ownerId, String eventId, String userToBeAd
     com.google.api.services.calendar.model.Event originalEvent = ownerCalendar.events().get("primary", eventId)
         .execute();
     List<EventAttendee> currentAttendees = originalEvent.getAttendees();
+    System.out.println(userToBeAddedEmail);
+    System.out.println(ownerId);
+    System.out.println(eventId);
```
Comment (rebekadittrich) [inline src/main/java/com/google/sps/servlets/Utils.java]: Is this left in from testing?

**Review by rebekadittrich — APPROVED** [review]

File: src/main/java/com/google/sps/servlets/Utils.java (hunk 1 above)
Comment (andru47) [reply src/main/java/com/google/sps/servlets/Utils.java]: Forgot to delete

File: src/main/webapp/add_event.html [hunk 2]
```diff
@@ -84,6 +84,9 @@
         <label for="event-location">Location</label>
         <input type="text" id="event-location" name="location">
 
+        <label for="isPublic">Is event public? (everyone can join)</label>
+        <input type="checkbox" id="isPublic" name="isPublic" checked="checked" style="width: auto;"><br>
```
Comment (LadyPython) [inline src/main/webapp/add_event.html]: maybe name it is-public and move style to CSS?

File: src/main/webapp/script.js [hunk 3]
```diff
@@ -93,16 +93,72 @@ function searchEvents() {
...
+    }
+  });
   document.getElementById('location-id').value = "all";
 }
 
-function showEvent(event) {
-  document.getElementById('events-container').innerHTML += '<div><h1>'
+function showEvent(event, alreadyJoined) {
+  console.log(alreadyJoined);
+  /*document.getElementById('events-container').innerHTML += '<div><h1>'
```
Comment (LadyPython) [inline src/main/webapp/script.js]: Delete this comment

**Review by LadyPython — CHANGES_REQUESTED** [review]

**Review by LadyPython — APPROVED** [review]

**Review by marcbury-g — APPROVED** [review]

**Pull request merged**